lib/sidekiq/form526_backup_submission_process/processor.rb

```diff
@@ -356,6 +352,10 @@ def get_form526_pdf
         }
       end
 
+      def get_form_from_external_api(headers, form_json)
+        EVSS::DisabilityCompensationForm::Service.new(headers).get_form526(form_json)
```

Comment:
This minor refactor should only make BDEX team's work easier when we migrate the underlying EVSS call, assuming BDEX team will follow it's usual flipper/provider pattern. So this looks good to me.